constrained-voucher.mkd

```diff
@@ -231,6 +231,16 @@ This consideration does not apply to the Registrar because:
 As per {{Section 3.6.1 of RFC7030}}, the Registrar certificate MUST have the Extended Key Usage (EKU) id-kp-cmcRA.
 This certificate is also used as a TLS Server Certificate, so it must also have the EKU id-kp-serverAuth.
 
+## TLS Client Certificates: IDevID authentication
+
+As described in {{Section 5.1 of RFC8995}}, the Pledge makes a connection to the Registrar using  TLS Client Certificate for authentication.
+
+Subsequently the pledge will send a raw voucher request.
+As explained below in {{registrar-identity}}, the "x5bag" may be used in the parboiled voucher request to communicate identity between registrar and MASA.
+
+The Pledge MUST NOT use the x5bag attribute in this way with the Registrar.
+A Registrar that processes a voucher request with an x5bag attribute MUST ignore it, and MUST use only the TLS Client Certificate extension for authentication.
+
```

Comment:
```suggestion
The Pledge SHOULD NOT use the x5bag attribute in this way with the Registrar.
A Registrar that processes a voucher request with an x5bag attribute MUST ignore it, and MUST use only the TLS Client Certificate extension for authentication.

A situation where the pledge MAY use the x5bag is for communications of certificate chains to the MASA. 
This would arise in some vendor specific situations involving outsourcing of MASA functionality, or rekey of IDevID certification authority.
```